requirements.txt

```diff
@@ -52,13 +52,6 @@ identify==1.5.11
     # via pre-commit
 idna==2.9
     # via requests
-importlib-metadata==3.3.0
-    # via
-    #   flake8
-    #   pluggy
-    #   pre-commit
-    #   pytest
-    #   virtualenv
```

Comment:
Do these changes matter?

Reply:
No, `importlib-metadata` is a backport of 3.8 stdlib functions which was needed on 3.7 but isn't on 3.8. (I didn't intentionally remove it, this was just pip-compile doing its thing.)